packages/core/src/browser/label-provider.ts

```diff
@@ -27,7 +27,7 @@ import { ResourceLabelFormatter, ResourceLabelFormatting } from '../common/label
 /**
  * @internal don't export it, use `LabelProvider.folderIcon` instead.
  */
-const DEFAULT_FOLDER_ICON = 'fa fa-folder';
+const DEFAULT_FOLDER_ICON = '';
```

Comment:
Can you confirm why we want the default folder icon to be non-existent?

Reply:
Because that's the way it works in vscode (default icon theme). I wanted to align with that.

Reply:
I believe this is for our custom `file-icons` theia theme, not the vscode theme:

_before_:

![image](https://user-images.githubusercontent.com/40359487/129364250-172b86c2-d9f1-4753-acc3-c0af1ea0fc00.png)

_after_:

![image](https://user-images.githubusercontent.com/40359487/129364333-20a5e513-85dd-458c-ba9e-861e5714c290.png)


I don't think we should break, if users want the vscode experience they would use `seti` (default in vscode):

![image](https://user-images.githubusercontent.com/40359487/129364429-d0f38e14-01e8-4845-b18e-363101f23aca.png)

Reply:
I see. Would you recommend to keep the old `fa` icon then? Or rather replace it with the codicon as well?

Reply:
Up to you, I think `codicon folder` would look nice :+1:, and perhaps in the future we can think of deprecating font-awesome in the framework altogther.

Reply:
Alright, replaced them with codicons 👍